pyomo/contrib/pynumero/doc/source/license.rst

```diff
@@ -0,0 +1,37 @@
+Copyright and License
+==========================
+
+Copyright 2018 Sandia Corporation. 
+Under the terms of Contract TODO with Sandia Corporation, 
```

Comment:
This copyright statement should be updated before we merge this. @jsiirola what should this be?